include/Core/UI/Component/AddonController.php

```diff
@@ -25,6 +25,55 @@ public function __construct( $file ) {
 	}
 
 	/**
+	 * Retrieves and outputs the Addon list HTML.
+	 *
+	 * return voids
+	 */
+	public function display_addons() {
+		// The products endpoint does not need a key or token to render published products.
+		// @todo: Change the API Url to get the actual addons.
+		$response = wp_remote_get( 'http://local.wordpress.dev/edd-api/products/' );
+
+		if ( is_wp_error( $response ) ) {
+			// @todo: Modify the error message to be displayed if required.
+			echo 'Error Found ( '. $response->get_error_message() .' )';
+		} elseif ( is_array( $response ) ) {
+			$body = wp_remote_retrieve_body( $response );
+			// Convert the JSON response to array
+			$result_set = json_decode( $body, true );
+			// The array key is set by the EDD plugin
+			if ( array_key_exists( 'products', $result_set ) ) {
+
+				$products = $result_set['products'];
+				foreach ( $products as $product ) : ?>
+
+					<div class="el-addon">
+						<h3 class="el-addon-title">
+							<?php _e( $product['info']['title'], 'email-log' ); ?>
+						</h3>
+
+						<a href="#" title="Starter Package">
+							<!-- @todo: Replace the thumbnail url from the $products array. -->
+							<img src="https://8333-presscdn-0-98-pagely.netdna-ssl.com/wp-content/themes/edd-v2/images/starter-package.png" class="attachment-showcase wp-post-image" alt="Alternate Title" title="Some Good Title" />
```

Comment:
Also, don't hotlink images from other servers. You should get the product image as part of the json